openstack/nova/templates/cell2-conductor-deployment.yaml

```diff
@@ -111,7 +111,7 @@ spec:
               subPath: logging.ini
               readOnly: true
             {{- include "utils.proxysql.volume_mount" . | indent 12 }}
-        {{- include "utils.proxysql.container" . | indent 8 }}
+        {{- tuple . .Values.rpc_workers | include "utils.proxysql.container" | indent 8 }}
```

Comment:
Why is this `rpc_workers` and not like the `conductor-deployment.yaml` `conductor.config_file.conductor.workers`?

Reply:
Ah, you are right. I'll fix it.